docs/reference/filter_field_definition.rst

```diff
@@ -6,49 +6,68 @@
 Filter field definition
 =======================
 
-These fields are displayed inside the filter box. They allow you to filter the list of entities by a number of different methods.
-
-A filter instance is always linked to a Form Type, there are 7 types available:
-
-* `Sonata\AdminBundle\Form\Type\Filter\NumberType`: display 2 widgets, the operator ( >, >=, <=, <, =) and the value,
-* `Sonata\AdminBundle\Form\Type\Filter\ChoiceType`: display 2 widgets, the operator (yes and no) and the value,
-* `Sonata\AdminBundle\Form\Type\Filter\DefaultType`: display 2 widgets, an hidden operator (can be changed on demand) and the value,
-* `Sonata\AdminBundle\Form\Type\Filter\DateType`: display 2 widgets, the operator ( >, >=, <= , <, =) and the value,
-* `Sonata\AdminBundle\Form\Type\Filter\DateRangeType`: display 3 widgets, the operator (between and not between) and the two values,
-* `Sonata\AdminBundle\Form\Type\Filter\DateTimeType`: display 2 widgets, the operator ( >, >=, <= , <, =) and the value,
-* `Sonata\AdminBundle\Form\Type\Filter\DateTimeRangeType`: display 3 widgets, the operator (between and not between) and the two values,
-
-The `Form Type` configuration is provided by the filter itself.
+These fields are displayed inside the filter box. They allow you to filter the list of entities by a number of different
+methods.
+
+A filter instance is always linked to a Form Type, there are 7 types available::
+
+* ``Sonata\AdminBundle\Form\Type\Filter\NumberType``:: display 2 widgets, the operator ( >, >=, <=, <, =) and the value;
+* ``Sonata\AdminBundle\Form\Type\Filter\ChoiceType``:: display 2 widgets, the operator (yes and no) and the value;
+* ``Sonata\AdminBundle\Form\Type\Filter\DefaultType``:: display 2 widgets, an hidden operator (can be changed on demand) and
+    the value;
```

Comment:
This line is causing problems

Reply:
Yes, this is the situation after my latest commit, but before, other lines were referenced as offending ones.
I've added many changes now trying to fix the build (replaced 1 backtick with 2 where missing, replaced 1 colon with 2 where missing, etc).

Reply:
I would recomend going back to your original changes and see what is failing together

Reply:
The minimum changes for what it's worth in this PR are working. Thank you.